s3_upload/s3_upload.py

```
@@ -174,7 +174,7 @@ def upload_single_run(args):
     )
 
 
-def monitor_directories_for_upload(config, dry_run):
+def monitor_directories_for_upload(config, dry_run) -> None:
```

Ruff (0.8.0)

`monitor_directories_for_upload` is too complex (14 > 10)

(C901)